model/messages/parsedmessage.go

```diff
@@ -0,0 +1,55 @@
+package messages
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-pg/pg/v10"
+	"go.opencensus.io/tag"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+
+	"github.com/filecoin-project/sentinel-visor/metrics"
+)
+
+type ParsedMessage struct {
+	Cid string `pg:",pk,notnull"`
+
+	From   string `pg:",notnull"`
+	To     string `pg:",notnull"`
```

Comment:
Would it be worth capturing the actor types for the `From` and `To` fields?

Reply:
`Method` implicitly allows determination of the type of the `To` address.
I haven't seen any cases where we've made use of the type of `From` addresses. Can you think of a use case for that?

Reply:
Off the top of my head the only case for `From` I can think of is differentiating multisig actors from account actors, but I don't know how frequently we will want that. I think this is fine as is.